README.md

```diff
@@ -138,11 +139,55 @@ To specify additional optional tool-specific parameters, see what flags are supp
 ```
 nextflow run main.nf --help 
 ```
+A more advanced nextflow command with optional parameters, where you can define the name of the output folder and provide an integer value for the number of genomic-interval files to be generated looks like this:
+```
+nextflow run main.nf --input samples.csv \
+                     --ref /path/to/ref.fasta --dict /path/to/ref.dict \
+                     --ponvcf /path/to/pon \
+                     --common_biallelic_variants /path/to/common_biallelic_variants \
+                     --outDir name_output_folder \
+                     --number_of_intervals INT
+```
```

Comment:
Remove this. I don't think we should be saying this is more advanced, your dot points below explain what each of the mandatory and optional parameters do.